internal/core/src/common/FieldMeta.h

```diff
@@ -267,6 +294,7 @@ class FieldMeta {
     FieldName name_;
     FieldId id_;
     DataType type_ = DataType::NONE;
+    DataType element_type_ = DataType::NONE;
```

Comment:
name: sub_element_type is better? my personal suggest

Reply:
The field element_type has already been added to proto, so for consistency, continue to call it this name.